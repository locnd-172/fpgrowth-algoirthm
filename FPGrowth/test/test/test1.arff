@relation retail

@attribute a {true}
@attribute b {true}
@attribute c {true}
@attribute d {true}
@attribute e {true}
@attribute f {true}
@attribute g {true}
@attribute h {true}
@attribute i {true}
@attribute j {true}
@attribute k {true}
@attribute l {true}
@attribute m {true}
@attribute n {true}
@attribute o {true}
@attribute p {true}
@attribute s {true}

@data

true,?,true,true,?,true,true,?,true,?,?,?,true,?,?,true,?
true,true,true,?,?,true,?,?,?,?,?,true,true,?,true,?,?
?,true,?,?,?,true,?,true,?,true,?,?,?,?,true,?,?
?,true,true,?,?,?,?,?,?,?,true,?,?,?,?,true,true
true,?,true,?,true,true,?,?,?,?,?,true,true,true,?,true,?
