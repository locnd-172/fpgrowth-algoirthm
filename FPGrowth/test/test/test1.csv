100,f
100,a
100,c
100,d
100,g
100,i
100,m
100,p
200,a
200,b
200,c
200,f
200,l
200,m
200,o
300,b
300,f
300,h
300,j
300,o
400,b
400,c
400,k
400,s
400,p
500,a
500,f
500,c
500,e
500,l
500,p
500,m
500,n
