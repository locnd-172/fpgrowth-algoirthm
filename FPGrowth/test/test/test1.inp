a c d f g i m p 
a b c f l m o 
b f h j o 
b c k p s 
a c e f l m n p 
