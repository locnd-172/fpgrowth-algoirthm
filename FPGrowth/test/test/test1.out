a:3
b:3
c:4
f:4
m:3
p:3
a c:3
a f:3
a m:3
c f:3
c m:3
c p:3
f m:3
a c f:3
a c m:3
a f m:3
c f m:3
a c f m:3
